INSTALL.linux.rst

```diff
@@ -0,0 +1,123 @@
+=====
+Linux
+=====
+
+
+  Tested on Arch Linux, Linux Mint 20, Ubuntu 20.04 and Fedora 33 (all amd64)
+
+
+Build Requirements
+------------------
+
+- cmake
+- make
+- ninja
+- gcc
+- g++
+- nasm
+- git
+- sudo
+- python (3.7+)
+- python development (python3-dev or python3-devel)
+- pip
+
+
+Arch
+^^^^
+
+.. code-block:: sh
+
+  sudo pacman -Sy
+  sudo pacman -S cmake make ninja gcc nasm git python3 python-pip
+
+
+Fedora
+^^^^^^
+
+.. code-block:: sh
+
+  sudo dnf update
+  sudo dnf install cmake make ninja-build gcc g++ nasm git python3-devel python3-pip
+
+
+Ubuntu
+^^^^^^
+
+.. code-block:: sh
+
+  sudo apt-get update
+  sudo apt-get install cmake make ninja-build gcc g++ nasm git python3-dev python3-pip
+
+
+Build Steps
```

Comment:
So, will leave specific comments below, but one general thing -- most of what's in these steps generally shouldn't be required for an end-user to install this library, so we should probably include the more "usual" install instructions. We're pushing wheels to PyPI: https://pypi.org/project/avif/#files which happens here: https://github.com/Julian/avif/blob/main/.github/workflows/packaging.yml so most users should not have to run any of the below.

Did you get something different when you installed purely via pip? I assume if you did, possibly because you wrote you're on ARM64, where yeah I didn't push ARM64 wheels -- I forget whether GH Actions offers ARM64 builders for Linux, I believe it actually does, and that if we enabled that you'd also get a wheel.

What you wrote up certainly is correct for a platform that doesn't currently have wheels being built for it, or to build a development build, then you'd go through the below to build from source, but yeah most users should just be able to run `pip install avif`. E.g. see below:

```
⊙  docker run --rm -it python:latest /bin/bash                                                                                                                          julian@Airm
root@32236ac1e544:/# python -m pip install avif
Collecting avif
  Downloading avif-0.3.0-cp39-cp39-manylinux2010_x86_64.whl (77 kB)
     |████████████████████████████████| 77 kB 2.4 MB/s
Collecting cffi>=1.0.0
  Downloading cffi-1.14.4-cp39-cp39-manylinux1_x86_64.whl (405 kB)
     |████████████████████████████████| 405 kB 11.7 MB/s
Collecting attrs
  Downloading attrs-20.3.0-py2.py3-none-any.whl (49 kB)
     |████████████████████████████████| 49 kB 5.8 MB/s
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 8.1 MB/s
Installing collected packages: pycparser, cffi, attrs, avif
Successfully installed attrs-20.3.0 avif-0.3.0 cffi-1.14.4 pycparser-2.20
WARNING: You are using pip version 20.3.3; however, version 21.0 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
root@32236ac1e544:/# python -c 'import avif'
```
```